clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -88,8 +88,9 @@ public class CommonClientConfigs {
         "If provided, the backoff per host will increase exponentially for each consecutive connection failure, up to this maximum. After calculating the backoff increase, 20% random jitter is added to avoid connection storms.";
 
     public static final String RETRIES_CONFIG = "retries";
-    public static final String RETRIES_DOC = "Setting a value greater than zero will cause the client to resend any request that fails with a potentially transient error." +
-        " It is recommended to set the value to either zero or `MAX_VALUE` and use corresponding timeout parameters to control how long a client should retry a request.";
+    public static final String RETRIES_DOC = "It is recommended to set the value to either `MAX_VALUE` or zero, and use corresponding timeout parameters to control how long a client should retry a request." +
```

Comment:
nit: We can change '\`' to `<code>`, so we can have syntax on document.

```suggestion
    public static final String RETRIES_DOC = "It is recommended to set the value to either <code>MAX_VALUE</code> or zero, and use corresponding timeout parameters to control how long a client should retry a request." +
```

before: 
<img width="628" alt="Screenshot 2025-04-30 at 3 24 22 PM" src="https://github.com/user-attachments/assets/0959bced-9f2f-4a1c-8e9e-2598f996f162" />

after:
<img width="634" alt="Screenshot 2025-04-30 at 3 22 56 PM" src="https://github.com/user-attachments/assets/0f6b12e6-fda5-4c28-b597-2dd7efbb081e" />